core/src/main/scala/org/ensime/core/DocResolver.scala

```diff
@@ -109,16 +94,16 @@ class DocServer(
     if (java) {
       val path = javaFqnToPath(sig.java.fqn)
       val anchor = sig.java.member.map { s =>
-        "#" + URLEncoder.encode(if (docType == Javadoc8) toJava8Anchor(s) else s, "UTF-8")
+        "#" + { if (docType == Javadoc8) toJava8Anchor(s) else s }
       }.getOrElse("")
-      s"http://localhost:${port.getOrElse(0)}/$jarName/$path$anchor"
+      s"$jarName/$path$anchor"
     } else {
       val scalaSig = maybeReplaceWithUsecase(jar, sig.scala)
       val anchor = scalaSig.fqn.mkString +
         scalaSig.member.map {
-          "@" + URLEncoder.encode(_, "UTF-8")
```

Comment:
yeah, I don't think we need to encode the links because all browsers do this automatically. Plus it makes it a lot easier to test.